### ISSUE TITLE
Bug: Mealplan link in side menu uses `CALENDAR_FIRST_DAY_OF_WEEK` settings to prefill `start` parameter, not `MEAL_PLAN_FIRST_DAY_OF_WEEK`

### ISSUE THREAD

**Issue by TurnrDev** [grocy/grocy#2286]
I have discovered a bug in which the link in the menu (on every page) for Meal Plan has the start parameter pre-filled with the first day of the calendar week instead of the first day of the meal plan week. Upon clicking in, in certain configurations, the meal plan page will show the previous week's plan.

Configuration under which I detected the bug:

```php
Setting('CALENDAR_FIRST_DAY_OF_WEEK', '0');
Setting('MEAL_PLAN_FIRST_DAY_OF_WEEK', '1');
```

As of writing this, in the timezone my grocy is configured to, the date of today is 2023-07-31, a Monday (day 1)
The link generated is `http://hostname:9283/mealplan?start=2023-07-30`, which changes to `http://hostname:9283/mealplan?start=2023-07-24` on page load and displays last week's meal plan.

Changing my configuration to make the settings match resolves the issue, which is fine for me, as I don't use the calendar anyway, but I just wanted to make you aware of the issue anyway!

**Comment by TurnrDev** [comment]
Yep! Looks like moment.js locale is set [here](https://github.com/grocy/grocy/blob/e946ec79d59f18b47b0ca3392348f4a677832fcd/public/js/grocy.js#L670) to be the calendar setting, but used [here](https://github.com/grocy/grocy/blob/e946ec79d59f18b47b0ca3392348f4a677832fcd/public/js/grocy.js#L783) under the context of the meal plan. I'm not sure what an ideal solution would be unless it's possible to have a second instance of moment, it might be to combine the settings.

**Comment by TurnrDev** [comment]
Lol, that's a solution I guess!

**Comment by berrnd** [comment]
> Lol, that's a solution I guess!

If that was a complain in the sense of you see only your referenced JS removed in that commit, please simply try things out before moaning, it works and the removed part was just unnecessary since a longer time.

**Comment by TurnrDev** [comment]
Nope, not a complaint! I just didn't think of that solution

**Comment by TurnrDev** [comment]
> Sorry for assuming the worst Jay! I should think better of people.

No problem @berrnd, not everybody is out to get ya :)
